### ISSUE TITLE
Engineer's Toolbox Galena Ore (Lead) not working in IC2 Classic Macerator

### ISSUE THREAD

**Issue by Amperthorpe** [buttilda/Another-One-Bites-the-Dust#103]
Other EngToolbox ores work fine.
IC2 support is enabled in the config.
The pack is stock Tekkit Legends.

**Comment by buttilda** [comment]
Then it mustn't be registered properly in the ore dictionary or it doesn't have an ingot

**Comment by Amperthorpe** [comment]
NEI shows it as working with other things, the the rock crusher, and that makes it into the dust from AOBD. It does have an ingot.

**Comment by buttilda** [comment]
Weird... can you send me the config file?
I don't have any special code added for that metal, there's no reason for it not to be working.

**Comment by Amperthorpe** [comment]
http://pastebin.com/VSb5APN6

**Comment by buttilda** [comment]
File looks alright. I'll have to download Engineer's Toolbox and test it myself

**Comment by Amperthorpe** [comment]
if you want to make sure it's the same, it's totally stock Tekkit Legends, right off the Technic Launcher.

**Comment by buttilda** [comment]
Downloaded IC2 and Engineer's Toolbox
It works... the macerator is processing the lead just as it should. So the problem is either due to the pack using an unstable version of either of the mods, some other mod somehow breaking it, or the pack has tweaked recipes and broken it. I can't think of any other reason why it could be happening

**Comment by Amperthorpe** [comment]
Oh, it's IC2 Classic, not normal IC2. That might be why.

**Comment by buttilda** [comment]
Could be. I used the latest IC2 as of now.
I didn't even know there was a 1.7.10 version of IC2 classic XD

**Comment by Xen1290** [comment]
https://github.com/TinyModularThings/IC2Classic

**Comment by Speiger** [comment]
Oh yeah. Reason is because IC2 Classic does not check for dusts.
It checks only the OreRegister Event... If the ore is nothing that it knows then it ignores that.
I can not support lead by myself but i will release soon a plugin that will support ores from other mods in IC2 Machines. Please give me some time. (I am working on 3 project at the same time)

**Comment by buttilda** [comment]
But I'm adding the recipe myself, should that be working?

**Comment by Speiger** [comment]
Jup that should work...
if you are using the API

**Comment by buttilda** [comment]
I use the API from IC2 Experimental, is it interchangeable with yours?

**Comment by Speiger** [comment]
IC2 Classic supports most parts of the Exp API. Based on what it has. (Example: Ore washing recipes will not work because classic does not have a orewasher) but everything else that IC2 Classic has is supportet in the Exp API. Also classic builds on top of the Exp API. Also has a maven... Check it out^^ (Wiki tells it)

**Comment by buttilda** [comment]
Oh i know what the problem is...

I have Lead blacklisted because it's natively supported by IC2 Experimental, but since (I'm guessing) it's not supported by Classic it ends up just being ignored.

Easy to fix! But, is there any way to tell which IC2 is running? Since the mod ids are the same, I don't know how to differentiate between them. @Speiger

**Comment by Speiger** [comment]
IC2Classic has a seperate ModID which loads with IC2Classic so that you can detect that. @ganymedes01 just check my api files (IC2Classic) in the Info folder there you find how i check which IC2C is loaded...

**Comment by Speiger** [comment]
@ganymedes01 i saw in a mail where you asked me something about the addition of ores in the IC2API but i can not find the issue anymore...

**Comment by buttilda** [comment]
I deleted it because I had the epiphany of what was causing the problem just after I posted it XD

**Comment by Speiger** [comment]
ok. From what i have seen it would have worked. But classic has only a macerator so most recipes would not be loaded (classic has fake lists in the recipes class to prevent crashes). So you should make differend recipes when classic is loaded

**Comment by buttilda** [comment]
Yeah I'll add specific support for Classic. Probably this weekend or next week :)

**Comment by Speiger** [comment]
ok. Classics newest update comes out on Sunday. When extactly no clue.

**Comment by buttilda** [comment]
Any possible breaking API changes I should be concerned about? I doubt I'll need to use anything other than the API.

**Comment by Speiger** [comment]
The final API is already on github install it and check out what has broken. Just classic class requires a update if you have it included. But everything else is just Support for addons. Like DrillAPI, and stuff like that.

**Comment by Revemohl** [comment]
Did this update ever come out? I have the latest versions of both AOBD and IC2C and yet I still can't macerate lead.

**Comment by Speiger** [comment]
@ganymedes01

**Comment by Speiger** [comment]
@ganymedes01 did this here got ever fixed?

**Comment by Revemohl** [comment]
Lead seems to be working just fine on IC2C's macerators now (without any need for AOBD, in fact),  they output any oredicted lead dust they find.
